wsmaster/che-core-api-auth/pom.xml

```diff
@@ -120,6 +116,11 @@
         </dependency>
         <dependency>
             <groupId>org.everrest</groupId>
+            <artifactId>everrest-core</artifactId>
```

Comment:
11:02:07 /home/codenvy/workspace/che-pullrequests-build/wsmaster/che-core-api-auth/src/main/java/org/eclipse/che/security/oauth/OAuthAuthenticator.java:234: error: cannot access JsonValue
11:02:07             return JsonHelper.fromJson(urlInputStream, userClass, null);
11:02:07                              ^
11:02:07   class file for org.everrest.core.impl.provider.json.JsonValue not found
11:02:07 1 error
11:02:08 [JENKINS] Archiving disabled